services/resources/resources/models.go

```diff
@@ -13,6 +13,9 @@ type Resource struct {
 	Description    string                     `json:"description"`
 	DefaultLevel   float64                    `json:"defaultLevel"`
 	Constraints    []ResourceConstraints      `json:"constraints,omitempty"`
+	LowerLimit     float64                    `json:"lowerLimit,omitempty"`
+	UpperLimit     float64                    `json:"upperLimit,omitempty"`
```

Comment:
don't we have the problem that null is interpreted as 0, and therefore we need an array of floats, or am I mixing this with another case?